data/campaigns/Eastern_Invasion/ART_MUSIC_LICENSE.txt

```diff
@@ -0,0 +1,307 @@
+
+AUDIO
+===============
+
+MUSIC
+---------------
+
+cry from elensefar (pluft) - CC BY-SA 4.0
+    https://forums.wesnoth.org/viewtopic.php?t=48565
+    https://soundcloud.com/pluft/cry-from-elensefar
+
+heavens (gabriel silver), remixed by dalas - CC BY-SA 4.0 (both)
+    https://forums.wesnoth.org/viewtopic.php?t=51483
+    https://www.youtube.com/watch?v=AOvwWOGH7bQ&list=PLk-SdLbgd5ol62NX0q7ys5ZbmJFVAMzCF&index=2
+
+
+SOUNDS
+---------------
+
+wind (hackerb9) - CC 0
+    edited by dalas; I release the edited sound under CC BY-SA 4.0
+    https://freesound.org/people/hackerb9/sounds/555246/
```

Comment:
The CC0 seems correct, but a quote from the page: "This is simply a mono version of Tony-B-kksm's "Gust of Wind, Moving Left to Right". See <https://freesound.org/people/Tony%20B%20kksm/sounds/80937/>". (The referenced work is also CC0, so that's ok).

Reply:
Based on my understanding, we can modify the license for CC0 specifically to CC BY-SA 4.0 (but not earlier versions of CC BY-SA to 4.0). I added Tony B kksm to the credits.

Reply:
CC0 is essentially a license agreement that gives the same rights as a work in the public domain, so I think you can release edits under literally any license you want. In other words, changing it to CC BY-SA 4.0 should be no problem.